.github/workflows/releaser.yml

```
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:
+      # so a vX.Y.Z-test1 doesn't trigger build
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

I would have expected to see a `$`

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+$'
```

Because otherwise the second regexp is useless, but also it lets me think this regexp is catching the -test tags you talk about

---

I thought it's a full match but given pre1,2 etc did build, probably not

---

it's weird and I'll admit, don't have time to figure out but... it works https://github.com/grol-io/grol/pull/51